src/RockPaperScissors/RockPaperScissors.sln

```diff
@@ -1,12 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-# VisualStudioVersion = 14.0.25420.1

```

Comment:
Looks like removing it entirely worked. We'll see when you eventually build with this change.